server/cli/launch.py

```diff
@@ -26,29 +26,29 @@
 
 def annotation_args(func):
     @click.option(
-        "--experimental-annotations",
+        "--disable-annotations",
         is_flag=True,
         default=False,
         show_default=True,
-        help="Enable user annotation of data.",
+        help="Disable user annotation of data.",
     )
     @click.option(
-        "--experimental-annotations-file",
+        "--annotations-file",
         default=None,
         show_default=True,
         multiple=False,
         metavar="<path>",
         help="CSV file to initialize editing of existing annotations; will be altered in-place. "
-        "Incompatible with --annotations-output-dir.",
+        "Incompatible with --annotations-dir.",
     )
     @click.option(
-        "--experimental-annotations-output-dir",
+        "--annotations-dir",
```

Comment:
Ideally we would also have test cases that execise --annotations-dir.    Also fine to leave for future work.